src/components/ui/ProductDetails.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'evergreen-ui';
+import QuantityBtn from './QuantityBtn';
+import CollapseBtns from './CollapseBtns';
+
+const ProductDetails = ({
+  productvalues,
+}) => {
+  const [productQuntity, setProductQuntity] = useState(1);
+  const onlyPositive = /[1-9]\d?/;
+
+  const validateNewQuantity = (prev) => {
+    if (onlyPositive.test(prev.target.value) || prev.target.value === '') {
+      setProductQuntity(prev.target.value);
+    }
+  };
+
+  const validateSubtraction = () => {
+    if (productQuntity > 1) {
+      setProductQuntity(productQuntity - 1);
+    }
+  };
+
+  const validateSum = () => {
+    if (productQuntity >= 0) {
+      setProductQuntity(productQuntity - 1 + 2);
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <div className="product-view-container">
+      <div className="product-img-container">
+        <div className="product-img-wrapper">
+          <img data-cy="product-image" alt="product-img" src={productvalues.image} />
+        </div>
+      </div>
+      <div className="product-info-container">
+        <div className="product-info-wrapper">
+          <h2 data-cy="product-info-title" className="product-info-title">{productvalues.title}</h2>
+          <p data-cy="product-info-price" className="product-info-price">
+            {`$${productvalues.price} USD`}
+          </p>
+          <form onSubmit={handleSubmit}>
+            <p>Quantity</p>
+            <QuantityBtn
+              validateNewQuantity={validateNewQuantity}
+              validateSubtraction={validateSubtraction}
+              validateSum={validateSum}
+              productQuntity={productQuntity}
+              setProductQuntity={setProductQuntity}
+            />
+            <div className="product-view-add-and-buy-btns">
+              <Button id="add-to-cart-btn">Add to cart</Button>
+              <Button appearance="primary" id="buy-product-btn">Buy it now</Button>
+            </div>
+          </form>
+          <p data-cy="product-info-description" className="product-info-description">{productvalues.description}</p>
+          <CollapseBtns productDetails={productvalues} />
+        </div>
+      </div>
+    </div>
+  );
+};
+
+ProductDetails.propTypes = {
+  productvalues: PropTypes.shape({
```

Comment:
should be in camelCase

Reply:
![image](https://user-images.githubusercontent.com/90296394/135180255-b2431e55-f259-4b9e-8142-75516f54cc6b.png)
 all productValues props were changed